graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -20,6 +20,41 @@
 )
 
 
+def _get_inherited_field_types(class_to_field_type_overrides, schema_graph):
+    """Return a dictionary describing the field type overrides in subclasses."""
+    inherited_field_type_overrides = dict()
+    for superclass_name, field_type_overrides in class_to_field_type_overrides.items():
+        for subclass_name in schema_graph.get_subclass_set(superclass_name):
+            inherited_field_type_overrides.setdefault(subclass_name, dict())
+            inherited_field_type_overrides[subclass_name].update(field_type_overrides)
+    return inherited_field_type_overrides
+
+
+def _validate_overriden_classes_have_no_superclasses(class_to_field_type_overrides, schema_graph):
+    """Assert that the fields we want to override are not defined in a superclass."""
```

Comment:
The name and the documentation talk of very different things. The documentation makes more sense.